crates/speedy-transform/src/web_transform/babel_import.rs

```diff
@@ -28,29 +28,24 @@ pub fn transformstyle(module: &mut swc_ecma_ast::Module, project_config: &Transf
     let item_index = module.body.iter().position(|citem| citem == item).unwrap();
     if let ModuleItem::ModuleDecl(ModuleDecl::Import(var)) = item {
       let source = &*var.src.value;
-      if let Some(childconfig) = config.iter().find(|&c| c.from_source == source) {
+      if let Some(child_config) = config.iter().find(|&c| c.from_source == source) {
         for specifier in &var.specifiers {
           match specifier {
             ImportSpecifier::Named(ref s) => {
-              let ident = format!("{}", s.local.sym);
+              let ident = s.local.sym.to_string();
```

Comment:
In this scenario use to_string directly is more idiomatic and also performant, because format is a macro which introduce some other formating code instead of just a function call